### PR TITLE
Adds elastic element to the right child element depending on GoCD version

### LINKED ISSUE #49: Elastic agents are broken in gocd 18.1.0
With gocd `18.1.0` released, one of the changes made was to [move the elastic profiles out of the server config](https://github.com/gocd/gocd/commit/e09c9f5d426d3197770557cfacc4060749a9be12). This breaking the elastic profiles being created with gomatic against gocd 18.1. The error I get trying to update my elastic profile:
```
File "/usr/lib/python3.6/site-packages/gomatic/go_cd_configurator.py", line 296, in save_updated_config
    self.__host_rest_client.post('/go/admin/restful/configuration/file/POST/xml', data, headers)
  File "/usr/lib/python3.6/site-packages/gomatic/go_cd_configurator.py", line 335, in post
    raise RuntimeError("Could not post config to Go server (%s) [status code=%s]:\n%s" % (url, result.status_code, message))
RuntimeError: Could not post config to Go server (http://localhost:8153/go/admin/restful/configuration/file/POST/xml) [status code=409]:
Invalid content was found starting with element 'elastic'. One of '{mailhost}' is expected.
```

I could move the elastic config to be out of the `<server>` tag but it would break for all the older gocd versions.

### PR DESCRIPTION
**Pull request by ayr-ton**
Fixes #49

### CODE REVIEW

**Comment by ayr-ton** [comment]
@akshaykarle @dudadornelles I'm not sure if these broken tests are related to the given commits.

**Comment by dudadornelles** [comment]
They seem to be @ayr-ton , tests are passing in master.

File: gomatic/go_cd_configurator.py [hunk 1]
```diff
@@ -258,7 +263,13 @@ def ensure_replacement_of_elastic(self):
 
     @property
     def elastic(self):
-        return Elastic(self.__server_element_ensurance().element.find('elastic'))
+        gocd_version_with_elastic_outside_server_tag = 18
+        gocd_major_version = int(self.server_version.split('.')[0])
+        if gocd_major_version < gocd_version_with_elastic_outside_server_tag:
+            elastic_element = Elastic(Ensurance(self.__xml_root).element.find('elastic'))
+        else:
+            elastic_element = Elastic(self.__server_element_ensurance().element.find('elastic'))
```
Comment (dudadornelles) [inline gomatic/go_cd_configurator.py]: It seems line 269 and 271 are swapped here? If I swap them, the tests pass.

Comment (ayr-ton) [reply gomatic/go_cd_configurator.py]: @dudadornelles You're right. I will amend this commit with the lines swapped. :)

**Comment by ayr-ton** [comment]
@dudadornelles You're right. I had swapped the lines on `@property` elastic and now the tests are passing.

**Comment by dudadornelles** [comment]
Merged, version 5.10 has it! Thanks @ayr-ton and @akshaykarle

**Pull request merged**